Framework/BranchSDK.h

```diff
@@ -30,14 +30,10 @@ FOUNDATION_EXPORT const unsigned char BranchSDKVersionString[];
 
 #import <BranchSDK/BranchDeepLinkingController.h>
 
-#if !TARGET_OS_TV
-// tvOS does not support these features
```

Comment:
So we'll just have to manually place a check in each class/function where we don't support tvOS?

Reply:
Yes. Apps trying to use it will get errors similar to - API not available in this scope.